Ladybird/Headless/Test.cpp

```diff
@@ -70,7 +70,7 @@ static ErrorOr<void> collect_dump_tests(Vector<Test>& tests, StringView path, St
             continue;
         }
 
-        if (!name.ends_with(".html"sv) && !name.ends_with(".svg"sv))
+        if (!name.ends_with(".html"sv) && !name.ends_with(".svg"sv) && !name.ends_with(".xhtml"sv))
```

Comment:
Wait what the heck - how did we used to support this 🤔 

https://github.com/LadybirdBrowser/ladybird/blob/master/Tests/LibWeb/Text/input/XHTML/script-execution.xhtml

Reply:
Maybe that test didn't even run? :thonk:

Reply:
Missed it since my steps were:

```
./Build/ladybird-debug/bin/headless-browser --run-tests Tests/LibWeb --rebaseline
```

Then realizing the test didn't run, so fixed the xhtml thing, and just did a:


```
./Build/ladybird-debug/bin/headless-browser --run-tests Tests/LibWeb -f innerhtml-01.txt --rebaseline
```

😅

Reply:
Okay, this latest CI error is just a flake